doc/resources/doc/sources/dev-guide.adoc

```diff
@@ -1,144 +1,203 @@
-+
-[source,shell]
+
+Additionally, if you're using ClojureScript, you will need to pass the `--cljs` flag to `../bin/rebel` so that it can integrate the REPL:
+
+[source,clojure]
 ----
-acme.api$ echo 'app_root_logger=acme.api' > dev/log_dev_app.properties
+edge/acme.api$ ../bin/rebel --cljs -A:dev:dev/build:build
+[Edge] Starting development environment, please wait…
+[Rebel readline] Type :repl/help for online help info
+[Edge] Loading Clojure code, please wait...
+[Edge] Now enter (go) to start the dev system
+dev=> 
 ----
-. Create `dev/dev.clj` with a boilerplate bringing in lib.app.dev
-+
+
+[[editor_clj_repl]]
+=== Editor REPLs
+
+Editor REPLs do not automatically place you in the `dev` namespace.
+Instead your REPL will look like so:
+
 [source,clojure]
 ----
-(ns dev
-  (:require
-    [dev-extras :refer :all]))
+user=>
 ----
-. Create a `.dir-locals.el` containing your dev jack-in aliases
-+
-[source,elisp]
+
+Here, you should type `(dev)` to get to the dev namespace:
+
+[source,clojure]
 ----
-((nil
-  (cider-clojure-cli-global-options . "-A:dev:build:dev/build")))
+user=> (dev)
+dev=>
 ----
 
-[[quick_start]]
-== Quick Start
+=== Emacs / CIDER
+
+CIDER has first-class support in edge.
+
+First you should open your `deps.edn` file.
+From here, you can do `M-x cider-jack-in-clj`.
+Your development aliases will be automatically picked up.
+
+To setup your Clojure REPL, follow the <<editor_clj_repl,Editor REPL instructions>>.
 
-Generally REPL session will start like so:
+[[nrepl_clojure]]
+=== nREPL
 
-[source,clojure-repl]
+`../bin/rebel` supports starting an nREPL.
+To do so, pass the `--nrepl` option before Clojure options to `../bin/rebel`:
+
+[source,shell]
 ----
-user=> (dev) ;; <1>
-dev=> (reset)
-:reloading (juxt.edge.demo.foo)
-:resumed
+acme.api$ ../bin/rebel --nrepl -A:dev
 ----
-<1> This is done automatically when using `../bin/rebel`
 
-[[usage]]
-== lib.app.dev reference
+This will start an nREPl on localhost at port 5600.
+The nREPL server will have cider loaded.
 
-=== `dev` functions and vars
+== Starting your system
 
-`dev`:: If you're in the `user` namespace, use this to swtich to `dev`.
-+
-[souce,clojure-repl]
+Edge uses the concept of a "system".
+A system holds your state (e.g. web servers or database connections) and things that need that state (e.g. resource handlers need database connections).
+
+From the `dev` namespace, type `(go)` to start the dev system.
+
+[source,clojure]
 ----
-user=> (dev)
+dev=> (go)
+[Edge] Website listening on: http://localhost:4040
+[Edge] Now make code changes, then enter (reset) here
+:initiated
 ----
-`reset`:: Use this to start the system, and after making code changes.  It will reload your code, and your system.
-+
-[source,clojure-repl]
+
+== Making code changes
+
+When developing Clojure will reload the code inside of the running process, instead of restarting the process.
+To do this, type `(reset)` at the REPL.
+
```

Comment:
Worth mentioning you can add `(cider-interactive-eval "(dev-extras/reset)")` to a keybinding to make this process easier in emacs